### PR TITLE
Implement file:writeln method to write new line

### PR DESCRIPTION
**Pull request by anjanagunasekera007**
Closes issue #2838 by implementing the writtln method

### CODE REVIEW

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/files/Writeln.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        try {
+            OutputStream outputStream = (BufferedOutputStream) destination.getNativeData("outStream");
+            if (outputStream == null) {
+                throw new BallerinaException("file is not opened in write or append mode:"
+                        + destination.getStringField(0));
+            }
+
+            outputStream.write(content);
+            outputStream.write(newLine);
+
```
Comment (lankavitharana) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/files/Writeln.java]: better if you can remove these extra spaces

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/files/Writeln.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                @Argument(name = "file", type = TypeEnum.STRUCT, structType = "File",
+                        structPackage = "ballerina.lang.files")},
+        isPublic = true
+)
+public class Writeln extends AbstractNativeFunction {
+
+    @Override public BValue[] execute(Context context) {
+
+        byte[] content = getBlobArgument(context, 0);
+        byte[] newLine = System.getProperty("line.separator").getBytes(Charset.defaultCharset());
```
Comment (hasithaa) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/files/Writeln.java]: Use System.lineSeparator() instread of "System.getProperty("line.separator")"

File: modules/ballerina-native/src/test/java/org/ballerinalang/nativeimpl/functions/FileTest.java [hunk 3]
```diff
@@ -309,6 +309,20 @@ public void testWrite() throws IOException {
         Assert.assertEquals(byteContent.blobValue(), getBytesFromFile(targetFile), "Written wrong content");
     }
 
+    @Test
+    public void testWriteNewLine() throws IOException {
+        String targetPath = "temp/writeln-file.txt";
+        File targetFile = new File(targetPath);
+        byte[] content = "Sample Text".getBytes();
+        byte[] newline = ("Sample Text" + System.getProperty("line.separator")).getBytes();
```
Comment (hasithaa) [inline modules/ballerina-native/src/test/java/org/ballerinalang/nativeimpl/functions/FileTest.java]: Use System.lineSeparator() instread of "System.getProperty("line.separator")"

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/files/Writeln.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        packageName = "ballerina.lang.files",
+        functionName = "writeln",
+        args = {@Argument(name = "blob", type = TypeEnum.BLOB),
+                @Argument(name = "file", type = TypeEnum.STRUCT, structType = "File",
+                        structPackage = "ballerina.lang.files")},
+        isPublic = true
+)
+public class Writeln extends AbstractNativeFunction {
+
+    @Override public BValue[] execute(Context context) {
```
Comment (djkevincr) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/files/Writeln.java]: Is this @Override annotation formatted correctly?

File: modules/ballerina-native/src/test/java/org/ballerinalang/nativeimpl/functions/FileTest.java [hunk 5]
```diff
@@ -309,6 +309,21 @@ public void testWrite() throws IOException {
         Assert.assertEquals(byteContent.blobValue(), getBytesFromFile(targetFile), "Written wrong content");
     }
 
+    @Test
+    public void testWriteNewLine() throws IOException {
+        String targetPath = "temp/writeln-file.txt";
+        File targetFile = new File(targetPath);
+        byte[] content = "Sample Text".getBytes();
+//        byte[] newline = ("Sample Text" + System.getProperty("line.separator")).getBytes();
```
Comment (hasithaa) [inline modules/ballerina-native/src/test/java/org/ballerinalang/nativeimpl/functions/FileTest.java]: Remove this commented code.

**Pull request merged**